application/modules/admin/views/scripts/shipment/ship-it.phtml

```diff
@@ -14,7 +14,7 @@
     </legend>
 
     <form method="post" action="" id="shipForm">
-        <div class="form-group" style="overflow-y: scroll;height: 400px;">
+        <div class="form-group" style="overflow-y: scroll;height: 400px;width: 1200px;">
```

Comment:
Please check formatting, indent should match the closing tag's position

Reply:
- Having a fixed height of 400px doesn't cater for all screen sizes and leaves a lot of unused screen space on larger monitors. This needs to be more responsive so that it uses the available screen area while always showing the search bar and the Enroll/Cancel buttons

- The overflow-x is defaulting to scroll in my browser. I would like to see horizontal overflow text being wrapped rather than causing a horizontal scrollbar to appear

![image](https://user-images.githubusercontent.com/3584031/111585002-550ba400-87c7-11eb-9ba0-dccf455f579c.png)

Reply:
Looking into!